permissions/Phone.permission

```diff
@@ -38,3 +38,7 @@ dbus-system.talk org.ofono.SmartMessagingAgent
 # BEG sessionbus-com.jolla.voicecall.ui.resource
 dbus-user.talk com.jolla.voicecall.ui
 # END sessionbus-com.jolla.voicecall.ui.resource
+
+# BEG sessionbus-com.jolla.settings
+dbus-user.call com.jolla.settings=com.jolla.settings.ui.*@/*
```

Comment:
Can this be triggered by any other process than voicecall-ui-jolla?

As this is quite specific for phone app, I am thinking that this should go to voicecall-ui.profile instead.

Reply:
Well the exact case I was here after is implemented in voicecall-jolla so in a way having the rule in .profile could indeed make sense. 

But actually there seems to be also a somewhat similar multisim settings page open call in components-telephony -> SimSelector and the inherited SimPicker. And that is used also in Sailfish.Messages and Sailfish.Contacts and don't know how far it spreads. 

So all in all not fully sure. Would it make sense if I move this into the profile now and scope this bug/fix to handle the special phone number, and the multisim case to be handled separately, somehow?